docs/learn-proxies.md

```diff
@@ -48,14 +67,80 @@ account designates a proxy account, but anonymous proxies are the opposite. The
 the proxy relationship is the proxy account and the new account is the primary. Use extreme care
 with anonymous proxies; once you remove the proxy relationship, the account will be inaccessible.
 
+![anonymous proxy](assets/proxy_anonymous_diagram.png)
+
+## Why use a Proxy?
+
+Proxies are great to use for specific purposes because they add in a layer of security. Rather than
+using funds in one sole account, smaller accounts with unique roles complete tasks for the main
+stash account. This drives attention away from the main account and to proxies.
+
+Anonymous proxies in particular are great for permissionless management. In this example below,
+there is a multisig with four different accounts inside. Two of the accounts, Alice and Bob have an
+anonymous proxy attached to it. In the case that the multisig wanted to add or remove Alice or Bob
+or even add in a new account into the anonymous proxy, the anonymous proxy would take care of that
```

Comment:
Is this true?  I didn't know they could do that

Reply:
Yup! Logan explained that to me =)